apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -67,6 +67,11 @@ ConversationView.prototype = {
     return messageNode && this.messageAccessors.parse(messageNode);
   },
 
+  waitForFullRendering: function(thread) {
+    var lastId = thread.messages[thread.messages.length - 1].id;
+    this.accessors.findMessage(lastId);
+  },
```

Comment:
__question:__ maybe something like this will be less timeout-risky and reliable enough?
```
var accessors = this.accessors;
this.client.waitFor(function() {
  return accessors.messages.length === thread.messages.length;
});
```